### ISSUE TITLE
achcli: add support for IAT Batches, ADV Entries, and more

### ISSUE THREAD

**Issue by vxio** [moov-io/ach#890]
ACH Version: `v1.6.3`

The `achcli` utility does not currently support ach files with IAT batches and ADV entries (and possibly other items). We should add support to print these items out to stdout.

**Comment by adamdecaf** [comment]
Example for IAT's description:

```

  BatchNumber  SECCode  ServiceClassCode     IATIndicator   DestinationCountryCode  FE Indicator   FE ReferenceIndicator   FE Reference   CompanyEntryDescription
  1            IAT      220 (Credits Only)                  US                      FV             DL12429418                             GAS BILL

    OriginatorIdentification       ISOOriginatingCurrencyCode    ISODestinationCurrencyCode  ODFIIdentification  EffectiveEntryDate   OriginatorStatusCode 
    123456789                      GBP                           USD                         98765432            190625 

    TransactionCode  RDFIIdentification  AccountNumber  Amount  AddendaRecords  TraceNumber    Category
    27               12345678            9184XXXX       10014   1               1489718971

      Addenda10 // if applicable 
      ...

      Addenda11 // if applicable 
      ...

    TransactionCode  RDFIIdentification  AccountNumber  Amount  AddendaRecords  TraceNumber    Category
    ... 

      Addenda10 // if applicable 
      ...

      Addenda11 // if applicable 
      ...

      Addenda12 // if applicable 
      ...

      Addenda13 // if applicable 
      ...

      Addenda14 // if applicable 
      ...

      Addenda15 // if applicable 
      ...

      Addenda16 // if applicable 
      ...

      Addenda17 // if applicable 
      ...

      Addenda18 // if applicable 
      ...

      Addenda98 // if applicable 
      ...
 
      Addenda99 // if applicable  
      ...
```

**Comment by susansubedi1** [comment]
Hey @adamdecaf  can I pick this up and try to hash it out?

**Comment by adamdecaf** [comment]
For sure! That would be great. Let me know if you need any help.

**Comment by wadearnold** [comment]
@adamdecaf  Looks like this should have been closed with #1100 ?
